hydroflow/src/lang/lattice/last_write_wins.rs

```diff
@@ -0,0 +1,99 @@
+use std::{cmp::Ordering, marker::PhantomData};
+
+use super::Convert;
+use crate::lang::lattice::{Compare, Lattice, LatticeRepr, Merge};
+
+pub struct LastWriteWins<M: Ord, T> {
+    _phantom: PhantomData<(M, T)>,
+}
+impl<M: Ord, T> Lattice for LastWriteWins<M, T> {}
+
+pub struct LastWriteWinsRepr<M: Ord, T> {
+    _phantom: PhantomData<(M, T)>,
+}
+impl<M: Ord + Clone, T: Clone> LatticeRepr for LastWriteWinsRepr<M, T> {
+    type Lattice = LastWriteWins<M, T>;
+    type Repr = (M, T);
+}
+
+impl<M: Ord + Clone, T: Clone> Merge<LastWriteWinsRepr<M, T>> for LastWriteWinsRepr<M, T> {
+    fn merge(
+        this: &mut <LastWriteWinsRepr<M, T> as LatticeRepr>::Repr,
+        delta: <LastWriteWinsRepr<M, T> as LatticeRepr>::Repr,
+    ) -> bool {
+        let mut changed = false;
+
+        let (marker_this, payload_this) = this;
+        let (marker_delta, payload_delta) = delta;
+
+        match marker_delta.cmp(marker_this) {
+            Ordering::Less => {
+                // do nothing.
+            }
+            Ordering::Equal => {
+                // If markers are equal then payloads must be equal
+                // But don't want to require T: PartialEq + Debug
+                // assert_eq!(*payload_this, payload_delta)
+                // Maybe an additional node-id tag could be added
+                // to ensure a total order for when the marker is tied?
```

Comment:
Should be ok to require that, does the test types have those traits? Also `debug_assert_eq!` will avoid prod runtime cost